src/main/kotlin/no/nav/familie/tilbake/api/DokumentController.kt

```diff
@@ -128,4 +132,23 @@ class DokumentController(
         lagreUtkastVedtaksbrevService.lagreUtkast(behandlingId, fritekstavsnitt)
         return Ressurs.success("OK")
     }
+
+    @Operation(summary = "Sjekker om periode er like og kan sammenslås")
+    @PostMapping(
+        "/sjekk-likhet-perioder/{behandlingId}",
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    @Rolletilgangssjekk(
+        Behandlerrolle.SAKSBEHANDLER,
+        "Sjekker om perioder er like - unntatt dato og beløp",
+        AuditLoggerEvent.UPDATE,
+        HenteParam.BEHANDLING_ID,
+    )
+    fun erPerioderLike(
```

Comment:
Finnes det en bedre plassering av denne? Burde den være i en annen controller, eventuelt opprettes ny?

Reply:
Tror ikke jeg ville forvente å finne dette endepunktet her neo. Kanskje ny kontroller om det ikke finnes noen som passer bedre?

Reply:
Er det meningen å utvide denne senere (sjekk på vilkår osv, eller ser dere for dere at dette blir nye endepunkt? ) 
Hvis denne bare skal sjekke perioder kunne den kanskje ligget i FaktaFeilutbetalingController??? 
Man ja, kanskje ny :D

Reply:
Egen controller eller behandling gir mening for min del. DokumentController virker ikke som "riktig" sted.

Reply:
> Er det meningen å utvide denne senere (sjekk på vilkår osv, eller ser dere for dere at dette blir nye endepunkt? ) Hvis denne bare skal sjekke perioder kunne den kanskje ligget i FaktaFeilutbetalingController??? Man ja, kanskje ny :D

Ikke FaktaFeilutbetaling - siden vilkår også må være like (etterhvert) så tenker jeg at vi bør løse de sjekkene som ett endepunkt, ikke flere...

Reply:
Tenkte ett endepunkt ja, men at sjekkene gjøres i hver sin service. 
Da lager jeg en ny controller hvertfall :D